dpd/p4/sidecar.p4

```diff
@@ -445,13 +564,26 @@ control NatIngress (
 				meta.l4_dst_port = hdr.icmp.data[31:16];
 			}
 		}
+
 		if (hdr.ipv4.isValid()) {
-			ingress_ipv4.apply();
+			if (meta.is_mcast && !meta.is_link_local_mcast)  {
```

Comment:
I think the bifurcations in this apply block are getting varied enough where we should consider having this be driven by a static table instead of all this if/else logic. Similar to the `Services` control.

Reply:
yep, i had something like that earlier with `const entries`, will do.

Reply:
`apply cannot be called from actions` for triggering a table apply (in an action) from const table, but I can make some of this nicer.